client/dashboard/app/locale/index.ts

```diff
@@ -8,5 +9,10 @@ type ComputedAttributes = {
 export function useLocale() {
 	const { user } = useAuth();
 	const u = user as typeof user & ComputedAttributes;
-	return u.localeVariant || u.localeSlug || user.locale_variant || user.language || 'en';
```

Comment:
Needed to only use this functionality so extracted it out to another util function.